examples/train/dspark_qwen3_4b_gsm8k_online.sh

```
@@ -0,0 +1,141 @@
+    --model "$MODEL" \
+    --data "$DATASET_FILE" \
+    --max-samples "$MAX_SAMPLES" \
+    --output "$OUTPUT_DIR" \
+    --seq-length "$SEQ_LENGTH"
+
+# Step 2: Launch vLLM server in the background
+echo "=== Step 2: Launching vLLM server ==="
+CUDA_VISIBLE_DEVICES="$VLLM_GPU" \
+ASCEND_RT_VISIBLE_DEVICES="$VLLM_GPU" \
+python scripts/launch_vllm.py "$MODEL" \
+    --target-layer-ids $TARGET_LAYER_IDS \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)